packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/CreateCategoriesTests.java

```diff
@@ -0,0 +1,223 @@
+                .assertThat()
+                    .field(FIELD_PARENT_ID).is(rootCategory.getId())
+                .assertThat()
+                    .field(FIELD_HAS_CHILDREN).is(false)
+                .assertThat()
+                    .field(FIELD_ID).isNotEmpty();
+
+        STEP("Create two categories under the previously created (as admin)");
+        final RestCategoryModel aSubCategory1 = new RestCategoryModel();
+        aSubCategory1.setName(RandomData.getRandomName("SubCategory"));
+        final RestCategoryModel aSubCategory2 = new RestCategoryModel();
+        aSubCategory2.setName(RandomData.getRandomName("SubCategory"));
+        final List<RestCategoryModel> categoriesToCreate = List.of(aSubCategory1, aSubCategory2);
+        final RestCategoryModelsCollection createdSubCategories = restClient.authenticateUser(dataUser.getAdminUser())
+                .withCoreAPI()
+                .usingCategory(createdCategory)
+                .createCategoriesList(categoriesToCreate);
+        restClient.assertStatusCodeIs(CREATED);
+        createdSubCategories.assertThat()
+                .entriesListCountIs(categoriesToCreate.size())
+                .getEntries()
+                    .get(0).onModel()
+                        .assertThat()
+                            .field(FIELD_NAME).is(aSubCategory1.getName())
+                        .assertThat()
+                            .field(FIELD_PARENT_ID).is(createdCategory.getId())
+                        .assertThat()
+                            .field(FIELD_HAS_CHILDREN).is(false)
+                        .assertThat()
+                            .field(FIELD_ID).isNotEmpty();
+        createdSubCategories.assertThat()
+                .entriesListCountIs(categoriesToCreate.size())
+                .getEntries()
+                    .get(1).onModel()
+                        .assertThat()
+                            .field(FIELD_NAME).is(aSubCategory2.getName())
+                        .assertThat()
+                            .field(FIELD_PARENT_ID).is(createdCategory.getId())
+                        .assertThat()
+                            .field(FIELD_HAS_CHILDREN).is(false)
+                        .assertThat()
+                            .field(FIELD_ID).isNotEmpty();
+        STEP("Get the parent category and check if it now has children (as regular user)");
+        final RestCategoryModel parentCategoryFromGet = restClient.authenticateUser(user)
+                .withCoreAPI()
+                .usingCategory(createdCategory)
+                .getCategory();
+
+        parentCategoryFromGet.assertThat()
+                .field(FIELD_NAME).is(aCategory.getName())
+                .assertThat()
+                .field(FIELD_PARENT_ID).is(rootCategory.getId())
+                .assertThat()
+                .field(FIELD_HAS_CHILDREN).is(true)
+                .assertThat()
+                .field(FIELD_ID).isNotEmpty();
```

Comment:
We could almost half the number of lines here by putting `assertThat...` on the same line as `field...` and IMO it would help the readability.

Reply:
Will do. Thx.